docs/design/deviceshifu/design-deviceShifu-rtsp.md

```diff
@@ -0,0 +1,93 @@
+# deviceShifu Rtsp overall design
```

Comment:
need to capitalize "RTSP", same for other occurrences

Reply:
This should allow us to connect to RTMP too right?

Reply:
maybe, just for now we can only allow use RTSP protocol